src/notBoth.js

```diff
@@ -0,0 +1,38 @@
+import { compose, complement, both } from 'ramda';
+
+
+/* eslint-disable max-len */
+/**
+ * A function which calls the two provided functions and returns the complement of `&&`ing the
+ * results.
+ * It returns true if the first function is false-y and the complement of the second function
+ * otherwise. Note that this is short-circuited, meaning that the second function will not be
+ * invoked if the first returns a false-y value. In short it will return true unless both predicates
+ * return true.
+ *
+ * In addition to functions, `RA.notBoth` also accepts any fantasy-land compatible
+ * applicative functor.
+ *
+ * @func notBoth
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.3.0|v2.3.0}
+ * @category Logic
+ * @sig (*... -> Boolean) -> (*... -> Boolean) -> (*... -> Boolean)
+ * @param {Function} f A predicate
+ * @param {Function} g Another predicate
```

Comment:
My last PR comment ;] 
Thank your for you patience and sorry for so much comments. Trying to keep the codebase consistent and top quality.

Insert `@see` tag to reference to original `both` function
```js
@see {@link http://ramdajs.com/docs/#both|both}
```

Reply:
Don't apologise. It's really helpful.